solana/Makefile

```diff
@@ -22,3 +25,10 @@ node_modules: package-lock.json
 clean:
 	anchor clean
 	rm -rf .anchor node_modules
+
+docker:
+	docker build -t solana-ntt-tests \
+		-f Dockerfile \
+		--build-arg SOLANA_CLI=$(SOLANA_CLI) \
+		--build-arg ANCHOR_CLI=$(ANCHOR_CLI) \
+		.
```

Comment:
Will this run at some point via tilt? This part might need to be duplicated in any Tiltfile that builds this as an image.

Reply:
The idea is definitely to integrate with Tilt. I added this for convenience but have no opposition to removing it. 
Compare with this Solana `Makefile` in the monorepo. https://github.com/wormhole-foundation/wormhole/blob/00f504ef452ae2d94fa0024c026be2d8cf903ad5/solana/Makefile#L91

This target should be doing the same thing, but could perhaps be merged with the `test` target.